LDAR_Sim/model_code/default_parameters/default_OGI_FU_parameters.py

```diff
@@ -0,0 +1,49 @@
+# ------------------------------------------------------------------------------
+# Program:     The LDAR Simulator (LDAR-Sim)
+# File:        default_OGI_FU_parameters
+# Purpose:     Default OGI FU parameters
+#
+# Copyright (C) 2018-2020  Thomas Fox, Mozhou Gao, Thomas Barchyn, Chris Hugenholtz
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the MIT License as published
+# by the Free Software Foundation, version 3.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# MIT License for more details.
+
+# You should have received a copy of the MIT License
+# along with this program.  If not, see <https://opensource.org/licenses/MIT>.
+#
+# ------------------------------------------------------------------------------
+
+default_OGI_FU_parameters = {
+    'version': '2.0',
+    'parameter_level': 'method',
+    'label': 'OGI_FU',
+    'module': 'dummy',
+    'deployment_type': 'mobile',
+    'measurement_scale': "component",
+    'sensor': 'OGI_camera',
+    'is_follow_up': False,
```

Comment:
Maybe I don't understand this parameter - but why is OGI_FU not considered follow-up?

Reply:
Thanks, corrected.